docs/source/machine-learning.rst

```diff
@@ -0,0 +1,96 @@
+Machine Learning
+================
+
+Dask facilitates machine learning, statistics, and optimization workloads in a
+variety of ways.  Generally Dask tries to support other high-quality solutions
+within the PyData ecosystem rather than reinvent new systems.  Dask makes it
+easier to scale single-machine libraries like Scikit-Learn where possible and
+makes using distributed libraries like XGBoost or Tensorflow more comfortable
+for average users.
```

Comment:
I would add that many of these require extra installations, detailed on their respective sites, and this might be a good place to start using whatever term you want for "dask-related" packages.